src/commands/link/link.js

```diff
@@ -276,16 +276,13 @@ const link = async (options, command) => {
     return exit()
   }
 
-  // If already linked to site. exit and prompt for unlink
   if (siteData) {
+    // If already linked to site. exit and prompt for unlink
     log(`Site already linked to "${siteData.name}"`)
     log(`Admin url: ${siteData.admin_url}`)
     log()
     log(`To unlink this site, run: ${chalk.cyanBright('netlify unlink')}`)
-    return exit()
-  }
-
-  if (options.id) {
+  } else if (options.id) {
```

Comment:
Sure! I removed `exit` because the `sites-create` tests were failing since it was calling `exit` even if everything was okay and made the test fail. By removing the `exit` function I needed to make sure the other paths of the `link` command weren't being executed, so I added the `else if` to the conditions 😅

Reply:
In general I would try to not change the code because of tests. 
As the exit function could be mocked in the tests and checked if it would be called.

But this is fine in this case